dexs/iziswap/index.ts

```diff
@@ -54,6 +55,11 @@ const adapter: SimpleAdapter = {
       start: () => getStartTimestamp(chains[CHAIN.BSC]),
       customBackfill: customBackfill(CHAIN.BSC as Chain, fetch)
     },
+    [CHAIN.ZKSYNC]: {
```

Comment:
hi, seem chainId 324 we set is era, could you change to `ERA` ?

Reply:
Sorry, my miss. Code updated